tests/unit/disco/components/TestAddon.js

```diff
@@ -386,25 +388,57 @@ describe(__filename, () => {
     });
   });
 
-  describe('<Addon type="theme"/>', () => {
-    let root;
+  describe.each([ADDON_TYPE_THEME, ADDON_TYPE_STATIC_THEME])(
+    `Addon for %s`,
+    async (type) => {
+      let root;
 
-    beforeEach(() => {
-      const data = { ...result, type: ADDON_TYPE_THEME };
-      root = renderAddon({ addon: data, ...data });
-    });
+      beforeEach(() => {
+        const data = { ...result, type };
+        root = renderAddon({ addon: data, ...data });
+      });
 
-    it('does render the theme image for a theme', () => {
-      expect(root.find('.theme-image')).toHaveLength(1);
-    });
+      it("doesn't render the logo for a theme", () => {
+        expect(root.find('.logo')).toHaveLength(0);
+      });
 
-    it("doesn't render the logo for a theme", () => {
-      expect(root.find('.logo')).toHaveLength(0);
+      it("doesn't render the description for a theme", () => {
```

Comment:
You might not need "for a theme" every time since it is grouped into a `describe` block that should mention the type of the addon.